packages/flutter/test/material/material_button_test.dart

```diff
@@ -2,12 +2,218 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../rendering/mock_canvas.dart';
+import '../widgets/semantics_tester.dart';
 
 void main() {
-  testWidgets('Default $MaterialButton meets a11y contrast guidelines', (WidgetTester tester) async {
```

Comment:
Any ideas why it was originally $MaterialButton? I'm not sure myself, but was curious about the change

Reply:
I don't know why we would need such a string interpolation here but I do know that it would throw an error when running the test:
`No tests match regular expression "^'Default \$MaterialButton meets a11y contrast guidelines'$".` 
The test passes fine without it.

Reply:
I have seen this in several tests. The reason for this is to guard against a class name change. If in the future `MaterialButton` was rename `FooMaterialButton` this test name string would not need to be updated. Practically speaking, it's not much of an issue as this is unlikely to be renamed, but that would be the reason.

Reply:
@darrenaustin thanks for the insight! I'll keep this in mind for other tests since this wouldn't be a big issue here.